pattern-1/README.md

```diff
@@ -0,0 +1,101 @@
+# DC/OS Resources for WSO2 API Manager Deployment Pattern 1
+
+## Quick Start Guide
+
+1. Install a DC/OS cluster on the local machine using the [DC/OS Vagrant](https://github.com/dcos/dcos-vagrant) 
+for evaluation purposes or prepare an existing DC/OS cluster for the deployment.
+
+2. Build WSO2 API Manager and WSO2 API Manager Analytics Docker images by 
+refering README.md files found in [dockerfiles/apim](../dockerfiles/apim) 
+and [dockerfiles/apim-analytics](../dockerfiles/apim-analytics) folders.
+
+3. Push above Docker images to a Docker registry or to the DC/OS nodes. If 
+a Docker registry is used, prefix the tags of Docker images with the Docker 
+registry hostname in the [DC/OS marathon-applications](marathon-applications/).
+
+4. Download [MySQL Connector for Java v5.1.34](https://downloads.mysql.com/archives/c-j/) 
+and copy its JAR file to the following folders:
+
+````bash
+cp path/to/mysql-connector-java-5.1.34-bin.jar pattern-1/volumes/apim/repository/components/lib/mysql-connector-java-5.1.34-bin.jar
+
+cp path/to/mysql-connector-java-5.1.34-bin.jar pattern-1/volumes/apim-analytics/repository/components/lib/mysql-connector-java-5.1.34-bin.jar
+````
+
+5. Mount a shared network volume to all DC/OS nodes and create four folders 
+for MySQL data, MySQL scripts, API-M, and API-M Analytics. If DC/OS Vagrant is used ssh 
+into each DC/OS node and create below folders on those hosts:
+
+````bash
+/volumes/wso2/mysql-data # a blank folder for persisting MySQL data
+/volumes/wso2/mysql-scripts # provides MySQL scripts for creating WSO2 API-M and API-M Analytics databases
+/volumes/wso2/apim # provides configurations, extensions, and shares runtime artifacts of the API-M containers
+/volumes/wso2/apim-analytics # provides configurations, and extensions to the API-M Analytics containers
+````
+
+6. Create a user group called ```wso2``` with the group id 200 on each DC/OS node 
+and change the group of above folder to ```wso2```:
+
+````bash
+groupadd --system -g 200 wso2
+chgrp -R wso2 /volumes/wso2
+````
+
+7. Copy the content found in the [volumes](volumes/) folder into the above folders.
```

Comment:
Once we copy, we would have to run the `chgrp -R wso2 /volumes/wso2` command again isn't it, because copying to` /volumes` would need to be done as root user, i.e.: `sudo`. 

That was my experience when running DC/OS on Vagrant, WDYT?

Reply:
A good catch! Yes, let me fix that in the README.md. Thanks!